docs/reference/data-streams/change-mappings-and-settings.asciidoc

```diff
@@ -546,10 +546,8 @@ The following get data stream API request retrieves information about
 GET /_data_stream/my-data-stream
 ----
 
-The API returns the following response. Note the `indices` property contains an
-array of the stream's current backing indices. The first item in the array
-contains information about the stream's oldest backing index,
-`.ds-my-data-stream-000001`.
+The response's `indices` property contains an array of the stream's current
+backing indices.
```

Comment:
Now that a backing index contains the date, we probably don't need to point out the oldest backing index.

Reply:
Now that we also have an API to migrate existing indices into data streams, there's no guarantee that all of a data stream's backing indices contain dates, so it's probably still useful to mention that the `indices` property contains an ordered list of backing indices.

Reply:
Good point. I re-added this info with 18d9cc6537516188587075cf676911bfdc6986ff.